.github/workflows/tinygo-base.yml

```diff
@@ -0,0 +1,27 @@
+name: tinygo-base
+
+on:
+  pull_request:
+    paths:
+      - .github/workflows/tinygo-base.yml
+      - builder/docker/tinygo/Dockerfile.base
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        platform: [linux/amd64, linux/arm64]
```

Comment:
The linux/arm64 will always timeout, but I think its worth leaving it here for the time being so that we catch early failures.

This workflow only runs on changes to the tinygo-base Dockerfile, or this action, in a pull-request.